msvc/capstone_static/capstone_static.vcxproj

```diff
@@ -1,6 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
On supported Visual Studio versions, I am fine to keep support of VS 2010. 

This commit, however, breaks it because it converts VS project file version from 2010 (v4) to 2015 (v14) disabling VS2010 for opening this project. 

To keep support of 2010, I propose to have a separate VS project file, say `capstone_static_wdk.vcxproj`, for driver support and reverting `capstone_static.vcxproj` to an older revision. Does this sound like a plan?

Alternative idea is to manually update `capstone_static.vcxproj` in the hope that VS2010 still works while VS2015 can covert it to the 2015 format with driver support. A challenge is that I do not have VS2010 and unable to test it, so does not appear to be the best idea.